lib/util/device-service-details.ts

```diff
@@ -19,6 +19,7 @@ export interface CurrentService {
 
 export interface CurrentServiceWithCommit extends CurrentService {
 	commit: string;
+	release_id: number;
```

Comment:
Would be nice extend the current `getWithServiceDetails` test case to also check for this new field.

Reply:
I've added a test to check that both `commit` and `release_id` exist on the services retrieved